packages/webview_flutter/webview_flutter_platform_interface/lib/src/v4/types/load_request_params.dart

```diff
@@ -0,0 +1,126 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+import 'package:webview_flutter_platform_interface/src/v4/webview_controller_delegate.dart';
+
+/// Defines the supported HTTP methods for loading a page in
+/// [WebViewControllerDelegate].
+enum LoadRequestMethod {
+  /// HTTP GET method.
+  get,
+
+  /// HTTP POST method.
+  post,
+}
+
+/// Extension methods on the [LoadRequestMethod] enum.
+extension LoadRequestMethodExtensions on LoadRequestMethod {
+  /// Converts [LoadRequestMethod] to [String] format.
+  String serialize() {
+    switch (this) {
+      case LoadRequestMethod.get:
+        return 'get';
+      case LoadRequestMethod.post:
+        return 'post';
+    }
+  }
+}
+
+/// Defines the parameters that can be used to load a page with the [WebViewControllerDelegate].
+///
+/// Platform specific implementations can add additional fields by extending this
+/// class and provide a factory method that takes the
+/// [LoadRequestParams] as a parameter.
+///
+/// {@tool sample}
+/// This example demonstrates how to extend the [LoadRequestParams] to
+/// provide additional platform specific parameters.
+///
+/// Note that the additional parameters should always accept `null` or have a
+/// default value to prevent breaking changes.
```

Comment:
This sentence applies to the example of extending the `LoadRequestParams` class in platform specific implementations and when doing so additional parameters should accept `null` or have a default value. I see it could have been written more clearly so I've rewritten it to (did the same for the similar classes `JavaScriptMessage`, `WebResourceError` and `WebViewControllerCreationParams`):

```dart
/// When extending [LoadRequestParams] additional parameters should always 
/// accept `null` or have a default value to prevent breaking changes.
```